kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/tools/jacoco/JacocoOnlineInstrumentation.kt

```diff
@@ -8,7 +8,7 @@ import java.io.*
 
 internal fun buildJvmAgentArgs(jarFile: File, binReportFile: File, excludedClasses: Set<String>): List<String> {
     val agentArgs = listOfNotNull(
-        "destfile=${binReportFile.canonicalPath},append=true,inclnolocationclasses=false,dumponexit=true,output=file,jmx=false",
+        "destfile=${binReportFile.canonicalPath},append=true,inclnolocationclasses=true,dumponexit=true,output=file,jmx=false",
```

Comment:
Shouldn't we enable this only if Robolectric was used? For normal projects, you likely do not need to include classes without locations

Reply:
All classes will not be included in the report anyway.
We don't know for sure whether Robolectric alone causes such an error or not.

It is quite non-trivial to understand before run whether such instrumentation occurred using Robolectric (because even analyze the dependencies of the Gradle project)